scripts/quests/jeuno/Path_of_the_Bard.lua

```diff
@@ -22,7 +22,6 @@ quest.reward =
     fame     = 30,
     fameArea = xi.quest.fame_area.JEUNO,
     gil      = 3000,
-    item     = xi.items.CHORAL_SLIPPERS,
```

Comment:
eh? I thought someone already did this lol

Reply:
https://github.com/AirSkyBoat/AirSkyBoat/pull/3180

Reply:
Like in a different PR somewhere?